src/com/cje/ResultView.java

```diff
@@ -0,0 +1,214 @@
+package com.cje;
+
+/**
+ * Created by Jieun Choi on 7/4/2016.
+ *
+ * class for result page view
+ */
+public class ResultView {
+    private static final String STYLE = "<style>"
+        + "table, th, td { border-collapse: collapse; border: 1px solid black; } "
+        + "th { padding: 8px; background-color: #99ccff; } "
+        + "td { padding: 5px; }"
+        + "body { font-size: 11px; }"
+        + "</style>";
+
+    private static final String HEADER = "<html><head>"+STYLE+"</head><body>";
+    private static final String TAIL = "</body></html>";
+    private static final String NO_DATA_MSG = "해당 년령<br>데이터 없음";
+
+    private float birthWeight;
+    private String birthWeightPercentile;
+    private float birthHeight;
+    private String birthHeightPercentile;
+    private float height;
+    private String heightPercentile;
+    private float weight;
+    private String weightPercentile;
+    private float head;
+    private String headPercentile;
+    private float bmi;
+    private String bmiPercentile;
+
+    private GrowthInfo gi_height;
+    private GrowthInfo gi_weight;
+    private GrowthInfo gi_head;
+    private GrowthInfo gi_bmi;
+
+    public ResultView() { initParam(); }
+
+    private void initParam() {
+        birthWeight = -1;
```

Comment:
I am questioning.
Do you really have specific logic to use these default values?
If not, this part just makes reader be confused.